.github/actions/presubmit/index.js

```diff
@@ -36,11 +36,13 @@ async function run() {
       });
       core.info(`Succeeded to remove label: ${LABEL}`);
     } catch (e) {
-      core.notice(`Failed to remove label: ${LABEL}`);
+      core.warning(`Failed to remove label: ${LABEL}`);
```

Comment:
This should still be an error, because it's been a failure in GitHub's API.

Reply:
An error will fail the overall workflow, no? Just cause we can't remove a tag doesn't mean that the overall status should be a fail if the other tests pass successfully.